test/elastic_record/integration/active_record_test.rb

```diff
@@ -48,7 +48,7 @@ def setup
     setup_project_database(
       'adapter'   => 'mysql2',
       'host'      => "localhost",
-      'database'  => 'elastic_record_test',
```

Comment:
Seems like a bad idea to name it the same as the database you're using for the rest of the tests, if you're dropping/creating it within specific testcases.